applications/tools/p2p-test/p2p-setup/src/main/kotlin/net/corda/p2p/setup/LinkManagerConfiguration.kt

```diff
@@ -83,6 +83,12 @@ class LinkManagerConfiguration : Callable<Collection<Record<String, Configuratio
     )
     var sessionsPerPeer = 4L
 
+    @Option(
+        names = ["--sessionRefreshThreshold"],
+        description = ["Session refresh threshold in milliseconds - *important* do not change"]
```

Comment:
```suggestion
        description = ["Session refresh threshold in milliseconds"]
```
I'd remove the last part now given it's configurable.